test_calculator.py

```diff
@@ -40,3 +57,39 @@ def test_div(self):
 
         result = calc.div(1., 1.e5)
         assert result == 1.e-5
+
+    def test_bitwise_or(self):
+        """Check 3 test cases which varies length of input for bitwise_or."""
+        calc = Calculator()
+        result = calc.bitwise_or(0b010000, 0b100000)
+        assert result == 0b110000
+
+        result = calc.bitwise_or(0b00011, 0b11000)
+        assert result == 0b11011
+
+        result = calc.bitwise_or(0b0010, 0b1000)
+        assert result == 0b1010
+
+    def test_xor(self):
+        """Test case for calculator's bitwise xor calculations."""
+        calc = Calculator()
+        result = calc.bitwise_xor(60, 13)
+        assert result == 49
+
+    def test_bitwise_not(self):
+        """Test cases for bitwise_not function in Calculator class.
+
+        To use:
```

Comment:
Please see [Google Style Python Docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html):

def example_generator(n):
    """Generators have a ``Yields`` section instead of a ``Returns`` section.

    Args:
        n (int): The upper limit of the range to generate, from 0 to `n` - 1.

    Yields:
        int: The next number in the range of 0 to `n` - 1.

    Examples:
        Examples should be written in doctest format, and should illustrate how
        to use the function.

        >>> print([i for i in example_generator(4)])
        [0, 1, 2, 3]

    """
    for i in range(n):
        yield i